src/pfe/portal/routes/templates.route.js

```diff
@@ -59,17 +57,19 @@ router.post('/api/v1/templates/repositories', validateReq, (req, res, _next) =>
     res.status(400).send("Invalid repository URL");
     return;
   }
-  if (!user.templates.addRepository(repositoryUrl, repositoryDescription)) {
+  try {
+    await user.templates.addRepository(repositoryUrl, repositoryDescription);
+  } catch (error) {
     res.status(400).send("Repository url already available.");
```

Comment:
I know this isn't your message, but shouldn't it say "Repository URL is already in use" or "already exists" or something like that? I don't know what "available" means here